db/index.js

```diff
@@ -0,0 +1,75 @@
+import { migrateWithPgClient as migrateEvaluateDB } from 'spark-evaluate/lib/migrate.js'
+import { migrateWithPgClient as migrateStatsDB } from '@filecoin-station/spark-stats-db-migrations'
+import pg from 'pg'
+
+const {
+// DATABASE_URL points to `spark_stats` database managed by this monorepo
+  DATABASE_URL = 'postgres://localhost:5432/spark_stats',
+
+  // EVALUATE_DB_URL points to `spark_evaluate` database managed by spark-evaluate repo.
+  // Eventually, we should move the code updating stats from spark-evaluate to this repo
+  // and then we won't need two connection strings.
+  EVALUATE_DB_URL = 'postgres://localhost:5432/spark_evaluate'
+} = process.env
+
+const poolConfig = {
+  // allow the pool to close all connections and become empty
+  min: 0,
+  // this values should correlate with service concurrency hard_limit configured in fly.toml
+  // and must take into account the connection limit of our PG server, see
+  // https://fly.io/docs/postgres/managing/configuration-tuning/
+  max: 100,
+  // close connections that haven't been used for one second
+  idleTimeoutMillis: 1000,
+  // automatically close connections older than 60 seconds
+  maxLifetimeSeconds: 60
+}
+
+const onError = err => {
+  // Prevent crashing the process on idle client errors, the pool will recover
+  // itself. If all connections are lost, the process will still crash.
+  // https://github.com/brianc/node-postgres/issues/1324#issuecomment-308778405
+  console.error('An idle client has experienced an error', err.stack)
+}
+
+export const getStats = async () => {
+  const stats = new pg.Pool({
```

Comment:
I find it confusing when the name does not mention the database. 

I think it's also important to distinguish between setting up a pool vs. a client.

```suggestion
export const getStatsPool = async () => {
  const statsPool = new pg.Pool({
```

The same comment applies to names related to the "evaluate" DB.

Reply:
After reading the rest of this PR, I think `getStatsPgPool` would be the best name.